### PR TITLE
added subsection for alternative start signal

### PR DESCRIPTION
**Pull request by sunava**
#### Description

This pull request addresses [Issue #890](https://github.com/RoboCupAtHome/RuleBook/issues/890).

**Proposed changes:**
- Clarifies scenarios in which teams may request an alternative start signal.
- Defines the procedure for handling such requests.

### CODE REVIEW

File: pages/general_rules/Procedure.tex [hunk 1]
```diff
@@ -91,34 +91,44 @@ \subsection{Removal of robots}
...
+    \end{itemize}
+    \textbf{Remark:} There may be penalties for using Ad-hoc start signals in some tests. The use of mouses, keyboards, and devices attached to ECRA computers is strictly forbidden.
+
+    \item \textbf{No penalties for physical constraints (HSR only):} 
+    In cases where a physical issue, such as a door bump, impedes the league's ability to ensure a standard start signal with full safety for the HSR robot, penalties will not be imposed. The following guidelines apply:
+    \begin{itemize}[nosep]
+        \item The team must document the physical issue and notify the committee prior to the Robot Inspection.
+        \item The issue must be reviewed and approved by the league's rules committee.
+        \item An alternative start method will be determined on-site in consultation with the committee.
+        \item Teams that bypass physical constraints by disabling safety features bear full responsibility for their actions.
```
Comment (LeroyR) [inline pages/general_rules/Procedure.tex:125]: Remove this (` \item Teams that bypass physical constraints by disabling safety features bear full responsibility for their actions.`). Has nothing to do with a guideline regarding physical constraints.

**Review by LeroyR — CHANGES_REQUESTED** [review]

File: pages/general_rules/Procedure.tex (hunk 1 above)
Comment (sunava) [reply pages/general_rules/Procedure.tex]: done

**Review by LeroyR — DISMISSED** [review]

**Review by johaq — DISMISSED** [review]

**Review by akinobu1998 — APPROVED** [review]

**Comment by hawkina** [comment]
the penelties for using alternative start signals: will have to be added to each task I presume? E.g. we could add a penelty to each score sheet with this PR or should that be another one?

**Comment by ARTenshi** [comment]
>the penelties for using alternative start signals

I think that the alternative start signal will depend on the arenas and will be decided by the TC/EC during the inspection days in case the arenas have any issues that prevent robots from entering (narrow access, bumps, or even not having a door) -- so, no penalties.

**Pull request merged**